core/src/main/java/io/undertow/server/HttpServerExchange.java

```diff
@@ -1950,7 +1950,7 @@ public void runResume() {
 
         private void invokeListener() {
             if(writeSetter != null) {
-                getIoThread().execute(new Runnable() {
+                super.getIoThread().execute(new Runnable() {
```

Comment:
It is based on findbugs report which shown, that getIoThread method exists in parent class and also in the outer class and it is suggested to clearly mark which one is used. Based on IDE the one from abstract class was actually used, this change just makes it clear.